### PR TITLE
DM-1819 Add missing config property

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
New default config added

### CODE REVIEW

File: content/protocol-integration/opcua-bundle/gateway-register.md [hunk 1]
```diff
@@ -209,6 +208,11 @@ gateway:
       401: MINOR
       1: WARNING
 
+  # Operation settings
+  operation:
+    # Controls if the OPC UA gateway performs an address space scan when it connects the first time to an OPC UA server
```
Comment (Oezge-Akin) [inline content/protocol-integration/opcua-bundle/gateway-register.md:213]: Can you add the detail that this is to define default behavior and value can be overwritten with OPC UA Server configuration?

**Review by Oezge-Akin — APPROVED** [review]

File: content/protocol-integration/opcua-bundle/gateway-register.md [hunk 2]
```diff
@@ -209,6 +208,11 @@ gateway:
       401: MINOR
       1: WARNING
 
+  # Operation settings
+  operation:
+    # Default behavior that controls if the OPC UA gateway performs an address space scan when it connects the first time to an OPC UA server. Can be overridden in the OPC UA Server config
```
Comment (rssag) [inline content/protocol-integration/opcua-bundle/gateway-register.md:213]:
```suggestion
    # Default behavior that controls if the OPC UA gateway performs an address space scan when it connects the first time to an OPC UA server. Can be overridden in the OPC UA Server config.
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions? @sag-eweingaertner

**Review by bggov — APPROVED** [review]

**Comment by sag-eweingaertner** [comment]
> what are the target versions? @sag-eweingaertner

10.17 -> auto-backport.

**Pull request closed without merging**